pkg/cmdimpl/status.go

```diff
@@ -15,22 +13,6 @@ import (
 
 // Status provides the implementation for the wego status application command
 func Status(allParams AddParamSet) error {
-	// verify the app is in the wego apps folder
```

Comment:
This removes the ability to let the user know the possibility of an unexcited app.